ui/about.go

```diff
@@ -6,21 +6,42 @@ package ui // import "miniflux.app/ui"
 
 import (
 	"net/http"
+	"sort"
 
+	"miniflux.app/config"
 	"miniflux.app/http/request"
 	"miniflux.app/http/response/html"
 	"miniflux.app/ui/session"
 	"miniflux.app/ui/view"
 	"miniflux.app/version"
 )
 
+type option struct {
+	Key string
+	Value string
+}
+
 func (h *handler) showAboutPage(w http.ResponseWriter, r *http.Request) {
 	user, err := h.store.UserByID(request.UserID(r))
 	if err != nil {
 		html.ServerError(w, r, err)
 		return
 	}
 
+	showOptions := user.IsAdmin
+	var options []*option
+	if showOptions {
+		keyValues := config.Opts.KeyValueMap()
+		keys := make([]string, 0, len(keyValues))
+		for key := range keyValues {
+			keys = append(keys, key)
+		}
+		sort.Strings(keys)
+		for _, key := range keys {
+			options = append(options, &option{Key:key, Value:keyValues[key]})
+		}
```

Comment:
You can do all the sorting in config package. This way it's also sorted in command line output.

Reply:
config package returns a map.
I do not think we can sort a map.

Reply:
Add a new function to config package.